src/components/AppBar.tsx

```diff
@@ -1,42 +1,63 @@
-import ShoppingBagIcon from "../icons/shoppingBag";
-import { ReactComponent as CrownIcon } from "../icons/crown.svg";
-import { ReactComponent as MenuIcon } from "../icons/menu.svg";
-import styles from "./appbar.module.scss";
-import { useState } from "react";
+import { ReactComponent as ShoppingBagIcon } from "../assets/icons/shopping_bag.svg";
+import { ReactComponent as CrownIcon } from "../assets/icons/crown.svg";
+import { ReactComponent as MenuIcon } from "../assets/icons/menu.svg";
+import "./appbar.styles.scss";
+import { Fragment, useState, useEffect } from "react";
+import { NavLink, Outlet, useNavigate } from "react-router-dom";
 
 function AppBar() {
   const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const navigate = useNavigate();
 
   const toggleMenu = () => {
     setIsMenuOpen(!isMenuOpen);
   };
 
+  const closeMenu = () => {
+    setIsMenuOpen(false);
+  };
+
+  useEffect(() => {
+    window.addEventListener("popstate", () => {
+      navigate(-1); // return to Home page when system back btn clicked
+    });
+  }, []);
+
   return (
-    <nav>
-      <CrownIcon className={styles.logo} />
+    <Fragment>
+      <nav>
+        <CrownIcon className="logo" />
 
-      <div className={styles.nav_container}>
-        <NavigationLinks />
+        <div className="nav_container">
```

Comment:
styling: 
```suggestion
        <div className="nav-container">
```
usually kebab-case in html tag attributes